src/rules/mod.rs

```diff
@@ -187,6 +189,7 @@ pub fn get_all_rules() -> Vec<Box<dyn LintRule>> {
     no_extra_non_null_assertion::NoExtraNonNullAssertion::new(),
     no_extra_semi::NoExtraSemi::new(),
     no_func_assign::NoFuncAssign::new(),
+    no_irregular_whitescpace::NoIrregularWhitespace::new(),
```

Comment:
`no_irregular_whitescpace` -> `no_irregular_whitespace`

Reply:
Oh thanks. Didn't notice that